sci-mathematics/cliquer/cliquer-1.23.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Can you make this EAPI=8 please? EAPI=7 is deprecated so pkgcheck complains. None of the EAPI 7->8 changes should affect this ebuild anyway.